account_payment_partner/README.rst

```diff
@@ -46,6 +46,8 @@ information, but there are some customers that don't want that everyone
 looking at the invoice sees the full account number (and even GDPR can say a
 word about that), so that's the reason behind the several options.
 
+This module adds "Payment mode" to Invoices Analysis.
```

Comment:
Put this in the list above (and you can take the occasion to correct the typo: `severals` > `several`